notsobot/notsobot.py

```diff
@@ -1530,12 +1610,16 @@ async def flipimg(self, ctx, urls: ImageFinder = None):
 
             def flip_img(b):
                 with Image.open(b) as img:
-                    img = ImageOps.flip(img)
+                    image = ImageOps.flip(img)
+                    img.close()
                 with BytesIO() as final:
-                    img.save(final, "png")
+                    image.save(final, "png")
                     file_size = final.tell()
                     final.seek(0)
-                    file = discord.File(final, filename="flip.png")
+                    filename = self.random_filename(True, "png")
+                    file = discord.File(final, filename=filename)
+                    image.close()
```

Comment:
This is unnecessary since `img` was created by the context manager which closes it automatically.